recipes/simbody/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conan import ConanFile, tools
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.scm import Version
+import os
+import textwrap
+
+
+class SimbodyConan(ConanFile):
+    name = "simbody"
+    description = "High-performance, open-source toolkit for science- and engineering-quality simulation"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/simbody/simbody"
+    topics = ("high-performance", "science", "simulation")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "CMakeLists.txt"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+         "build_lapack": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "build_lapack": True
+    }
+
+    def validate(self):
+        if self.options.build_lapack:
+            raise ConanInvalidConfiguration(
+                "build simbody from sources locally with default_options \"openblas:build_lapack\": True"
+            )
+
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    @property
+    def _module_file_rel_path(self):
+        return os.path.join("lib", "cmake", f"conan-official-{self.name}-variables.cmake")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("openblas/0.3.17")
+        self.requires("opengl/system")
+
+    def source(self):
+        tools.files.get(self, **self.conan_data["sources"][self.version],  
+                    destination=self._source_subfolder, strip_root=True)
+
```

Comment:
```suggestion
    def layout(self):
        cmake_layout(self, src_folder="Simbody/src")

```
Prefer cmake layout than the old cmake wrapper.